src/protocol/attribution/accumulate_credit.rs

```diff
@@ -264,6 +265,8 @@ mod tests {
 
     #[tokio::test]
     pub async fn accumulate() {
+        logging::setup();
+
```

Comment:
Crazy idea: we could move this to make_world().  I assume that there is no harm in doing it more than once (?)

Reply:
I love this suggestion. yes `logging::setup()` is safe to call in parallel, more than once, etc.